content/300-guides/200-deployment/110-deployment-guides/300-deploying-to-vercel.mdx

```diff
@@ -138,10 +138,28 @@ In order to enable Preview deployment for pull requests, create a separate datab
 
 > **Note**: because each pull request shares the same preview database, if you have multiple pull requests with diverging schema migrations, you will encounter problems applying them.
 
+## <inlinecode>vercel/pkg</inlinecode> <inlinecode>ENOENT</inlinecode> error
+
+If you use [vercel/pkg](https://github.com/vercel/pkg) to package your Node.js project, then you might encounter an error like the following at the bundle stage:
```

Comment:
Is the 'bundle stage' discussed elsewhere? Can't see it higher in the page...

Reply:
Good point. Thinking about this some more, we just mean "when you use vercel/pkg to package your project". That's implicit in the text already, so I've trimmed it down.